### PR TITLE
Code changes to add a comment line to Runner.java

### PR DESCRIPTION
**Pull request by ahmetya**
This changes are made to indroduce git feautires.

### CODE REVIEW

File: src/main/java/com/bootcamp/Runner.java [hunk 1]
```diff
@@ -12,7 +12,8 @@ public static void main(String[] args) throws InterruptedException {
         Helper help = new Helper();
         help.callTest();
 
-        // comment for this line
+        // comment for this line ######## now changed
```
Comment (ahmetya) [inline src/main/java/com/bootcamp/Runner.java]: Ahmet comments should start with capital letter please rewrite this comment.

**Comment by sirinevrim** [comment]
ok, perfect!

File: src/main/java/com/bootcamp/Runner.java (hunk 1 above)
Comment (ahmetya) [reply src/main/java/com/bootcamp/Runner.java]: Ok done

Comment (ahmetya) [reply src/main/java/com/bootcamp/Runner.java]: Ok changed and commited with amend.

**Comment by ahmetya** [comment]
Everything seems ok

**Pull request merged**